docs/content/concepts/configuration/config-schema.mdx

```diff
@@ -15,69 +15,75 @@ Run configuration allows providing parameters to jobs at the time they're execut
 
 A common use of configuration is for a [schedule](/concepts/partitions-schedules-sensors/schedules) or [sensor](/concepts/partitions-schedules-sensors/schedules) to provide configuration to the job run it is launching. For example, a daily schedule might provide the day it's running on to one of the ops as a config value, and that op might use that config value to decide what day's data to read.
 
 Dagster includes a system for gradually-typed configuration schemas. These make it easy to catch configuration errors before job execution, as well as to learn what configuration is required to execute a job.
 
-## Using Configuration Inside an Op
+## Defining and Accessing Configuration for an Op, Asset, or Resource
 
-This example shows how to write an op whose behavior is based on values that are passed in via configuration:
+Configurable parameters accepted by an op, asset, or resource are specified by providing a `config_schema` to the corresponding decorator. The structure of a `config_schema` is very flexible and [fully documented in the API Reference](/\_apidocs/config). However, most of the time you will want to provide a Python dictionary, with keys the names of parameters and values the types of those parameters.
 
-```python file=/concepts/configuration/configurable_op.py
-from dagster import job, op
+During execution, the specified parameters are accessible within the body of the op/asset/resource under `context.op_config` (for ops/assets) or `context.resource_config` (for resources). It might seem confusing that asset config is accessed under `context.op_config` instead of `context.asset_config`. However, assets are wrappers for ops, so when we access asset config we are literally just accessing config for the underlying op.
 
+Below we define a simple op and asset with identical `config_schemas` defining a single configurable parameter, `person_name`, as well as a resource with a configurable `url` parameter:
 
-@op
-def uses_config(context):
-    for _ in range(context.op_config["iterations"]):
-        context.log.info("hello")
+```python file=/concepts/configuration/configurable_op_asset_resource.py startafter=start endbefore=end
+@op(config_schema={"person_name": str})
+def op_using_config(context):
+    return f'hello {context.op_config["person_name"]}'
 
 
-@job
-def config_example():
-    uses_config()
-```
+@asset(config_schema={"person_name": str})
+def asset_using_config(context):
+    # Note how asset config is also accessed with context.op_config
+    return f'hello {context.op_config["person_name"]}'
 
-## Providing Run Configuration
 
-How you specify config values depends on how you're running your job:
+@resource(config_schema={"url": str})
+def resource_using_config(context):
+    return MyDatabaseConnection(context.resource_config["url"])
+```
 
-### Python API
+<Note>
```

Comment:
Markdown isn't usable inside these callouts, unfortunately. Could you change the backticks here to `<code></code>` so it displays correctly?